src/Context/SoapContext.php

```diff
@@ -83,6 +83,16 @@ public function iSendRequestWithParams($function, TableNode $params)
     {
         $this->sendRequest($function, [$params->getRowsHash()]);
     }
+    
+    /**
+     * Send SOAP request with params as nested array.
+     *
+     * @Given I call SOAP function :function with params as json:
+     */
+    public function iSendRequestWithNestedParamsArray($function, PyStringNode $json)
+    {
+        $this->sendRequest($function, [json_decode('{' . $json . '}', 'true')]);
```

Comment:
You are definitely correct. It's a typo.
Should remove single quotes. Still it works as it just thanks to PHP lazy type behavior.